.github/workflows/ci.yml

```diff
@@ -147,11 +147,18 @@ jobs:
       - name: "Build"
         run: nix develop -L --no-update-lock-file --command just build-cli
 
-      - name: "Test"
+      - name: "CLI Tests"
         env:
           RUST_BACKTRACE: 1
         run: nix develop -L --no-update-lock-file --command just test-cli
 
+      - name: "Functional Tests"
+        if: ${{ github.event_name == 'merge_group' || github.event_name == 'workflow_dispatch' }}
+        env:
+          RUST_BACKTRACE: 1
+          FLOXEM_FLOXTEST_TOKEN: "${{ secrets.FLOXEM_FLOXTEST_TOKEN }}"
+        run: nix develop -L --no-update-lock-file --command just functional-tests
```

Comment:
Ping @aakropotkin as you did https://github.com/flox/flox/commit/b3e6a0043da7b17491fc0a78a0286b1ea4e98b71